### DISCUSSION TITLE
When can i use your dropdown picker.

### DISCUSSION THREAD

**Discussion by SpringSan** [AbdullahQureshi1080/superforms-rn#12] [General]
Your project is very good. I wish use the dropdown picker. It's useful for me.

**Comment by AbdullahQureshi1080** [comment]
> Your project is very good. I wish use the dropdown picker. It's useful for me.

Thanks. I hope it helps. 

I'm working on form image picker at the moment, I can move to dropdown picker next. 

Will try to add the dropdown picker In the next two releases. 

Also PRs are welcome, if you wish to helpout.

**Reply by SpringSan** [reply]
That's great.
